src/redux/news/news.sagas.js

```diff
@@ -32,47 +32,50 @@ const {
   SUCCESS_DELETE_STATUS,
   SUCCESS_UPDATE_STATUS
 } = config.statuses;
-const {errorsLanguage} = config;
+const { errorsLanguage } = config;
 
 function* handleNewsLoad() {
   try {
-    yield put(setLoading(true));
+    yield put(setNewsLoading(true));
     const news = yield call(getAllNews, null);
-    yield put(setNews(news));
-    yield put(setLoading(false));
+    yield put(setNews(news.data.getAllNews));
+    yield put(setNewsLoading(false));
   } catch (error) {
     yield call(handleNewsError, error);
   }
 }
 
 function* handleArticleLoad({ payload }) {
   try {
-    yield put(setLoading(true));
+    yield put(setNewsLoading(true));
     const newsArticle = yield call(getArticleById, payload);
-    if (newsArticle.author) {
-      yield put(setArticle(newsArticle));
+    if (newsArticle.data.getNewsById.author) {
```

Comment:
Please, try to refactor this code to throw an exception inside the `getArticleById` and then handle it in saga inside the `catch` block.

Reply:
`yield call(getArticleById, payload)` should return created entity, but not the actual error, because if an error happens (you have `message` property with error messages or some other happens) we immediately go to the `catch` block

Reply:
Changed!